visual-js/visual-snapshots/src/commands/options.ts

```diff
@@ -80,6 +80,19 @@ export const suiteNameOption = new Option(
   "The name of the suite you would like to appear in the Sauce Visual dashboard."
 );
 
+export const testNameOption = new Option(
+  "--test-name <test-name>",
+  "The name of the test you would like to appear in the Sauce Visual dashboard." +
+    EOL +
+    "Supports the following parameters: {filename}"
+).default("{filename}");
+
+export const snapshotNameOption = new Option(
+  "--snapshot-name <snapshot-name>",
+  "The name of the snapshot you would like to appear in the Sauce Visual dashboard." +
+    EOL +
+    " Supports the following parameters: {filename}, {page}"
+).default("{filename}-{page}");
```

Comment:
I left this in `pdf.ts` because `{page}` may not make sense outside of PDF. Do you think it's better to keep it here?

Reply:
I moved it because I needed to import it in worker and I didn't want to import it from here.